client/offchain/Cargo.toml

```diff
@@ -27,14 +26,14 @@ sc-network = { version = "0.8", path = "../network" }
 sc-keystore = { version = "2.0.0", path = "../keystore" }
 
 [target.'cfg(not(target_os = "unknown"))'.dependencies]
-hyper = "0.12.35"
-hyper-rustls = "0.17.1"
+hyper = { version = "0.13.1", default-features = false, features = ["stream"] }
```

Comment:
I think you're missing features here anyway, as `hyper` needs the `tcp` or `runtime` feature in order to use tokio.

Reply:
You're right, I originally copied it from `grafana-data-source` but forgot to change it, meanwhile `hyper-rustls` enabled the `hyper/runtime` via https://github.com/ctz/hyper-rustls/blob/f51987a477af6d104783dee3b354e3db244f61a4/Cargo.toml#L29.

I'll fix it to pull all default features here for clarity.